src/encoded/docs/schema-changes.md

```diff
@@ -154,44 +154,104 @@ This new object is an array of example objects that can successfully POST agains
+                    "shearing (Covaris generic)",
+                    "shearing (Covaris LE Series)",
+                    "shearing (Covaris S2)",
+                    "shearing (generic)",
+                    "sonication (Bioruptor generic)",
+                    "sonication (Bioruptor Pico)",
+                    "sonication (Bioruptor Plus)",
+                    "sonication (Bioruptor Twin)",
+                    "sonication (Branson Sonifier 250)",
+                    "sonication (Branson Sonifier 450)",
+                    "sonication (generic microtip)",
+                    "sonication (generic)",
+                    "sonication (Sonics VCX130)"
+               ]
+
+            }
+        }
 
-1. In the **schemas** directory, increment the schema version (1->2), if it meets the above rule: 
+2. In the **schemas** directory, edit the existing properties in the corresponding JSON file named after the object and increment the schema version. For example if the original schema version for the library object being modified was "8", change it to "9" (8->9): 
         
         "schema_version": {
-            "default": "2"
+            "default": "9"
         }
 
-2. In the **upgrade** directory add an ```upgrade_step``` to an existing/new python file named after the object. An example upgrades the object from schema version (2->3):
+2. In the **upgrade** directory add an ```upgrade_step``` to an existing/new python file named after the object. An example to the upgrade step is shown below. Continuing with our example, after the upgrade all the existing "fragmentation methods" property must now be converted to list objects. The upgrade step ensures that the pre-existing library objects would now validate to the new schema. And since the schema is changing from version 8 to 9 the def must specify this (8->9):
+
+        @upgrade_step('library', '8', '9')
+        def library_8_9(value, system):
+            if 'fragmentation_method' in value:
+                value['fragmentation_methods'] = [value['fragmentation_method']]
+                value.pop('fragmentation_method')
+
+3. In the **tests/data/inserts** directory, we will need to change all the corresponding objects to follow the new schema. Continuing with our example, all fragmentation methods must now be converted to a list object. So, we need to find every instance of fragmentation methods in the inserts and convert them accordingly. For example:
 
-        @upgrade_step('{metadata_object}', '2, '3')
-        def {metadata_object}_2_3(value, system):
-            # {Redmine Issue}
+        #Schema version 8
+        "fragmentation_methods": "sonication (Bioruptor Twin)",
 
-            # lowercase all values in property 1
-            if 'property_1' in value:
-                value['property_1'] = value['property_1'].lower()
+        #Owing to schema version 9, the above should be changed to:
+        "fragmentation_methods": ["sonication (Bioruptor Twin)"],
```

Comment:
it has to be in the context of what is being changed in the inserts. You have to say it is library inserts that are being changed.